### ISSUE TITLE
Wpa3

### ISSUE THREAD

**Issue by axeldog** [ZerBea/hcxdumptool#467]
Can hcxdumptool attack wpa3?

**Comment by ZerBea** [comment]
No. There is no need to attack WPA3 SAE or to convert WPA3 SAE hashes because neither hashcat nor JtR can recover the PSK of  WPA3 SAE.
But hcxdumptool/hcxlabtool can downgrade a WPA3 SAE client to WPA2 if it is running in transition mode (WPA2/WPA3 SAE).
https://documentation.meraki.com/MR/Wi-Fi_Basics_and_Best_Practices/WPA3_Encryption_and_Configuration_Guide#WPA3_Transition_Mode

BTW:
I moved this to discussions, because it is not an issue.
